core/src/main/java/bisq/core/api/model/MarketDepthInfo.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.core.api.model;
+
+import lombok.AllArgsConstructor;
+
+import lombok.ToString;
+
+@ToString
+@AllArgsConstructor
+public class MarketDepthInfo {
+    public final String currencyCode;
+    public final Double[] buyPrices;
+    public final Long[] buyDepth;
```

Comment:
The depth should be a whole number since it represents the number of offers at a given price (correct me if I'm wrong on that, that's my understanding).  This is also based on the definitions outlined in the instructions (#140).  

The instructions say BigInt[] for the return type, however based on [this discussion](https://github.com/haveno-dex/haveno/issues/140#issuecomment-983157698) we are using int64 in the protobuf definitions.  That means that the generated java files are using long (as per [protobuf rules](https://developers.google.com/protocol-buffers/docs/overview#scalar)), and thus it's easier to just use Long in this java file as well.